lib/byron/test/integration/Test/Integration/Byron/Scenario/API/Transactions.hs

```diff
@@ -367,6 +373,67 @@ scenario_TRANS_CREATE_04d = it title $ \ctx -> do
   where
     title = "TRANS_CREATE_04 - Wrong password"
 
+
+scenario_TRANS_CREATE_08
+    :: forall (n :: NetworkDiscriminant) t.
+        ( DecodeAddress n
+        , EncodeAddress n
+        , PaymentAddress n ByronKey
+        )
+    => (Context t -> IO ApiByronWallet)
+    -> SpecWith (Context t)
+scenario_TRANS_CREATE_08 fixtureSource = it title $ \ctx -> do
```

Comment:
Do we really need a new scenario for this? Can't we simply add an assertion to an existing one :thinking: ?

Reply:
we can of course, but I thought it would be worth having dedicated minimal test to showing this (in the name of principle to have each test focusing on testing one thing)

Reply:
on the other hand `depth` expectations are not huge thing. and probably we can sneak it in other test.

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1583/commits/49bc06d30aa1c79a988c2f5cb8aceb411907bc7f